actions/run/src/run.ts

```diff
@@ -33,30 +35,40 @@ export async function run(): Promise<void> {
     args.push(...flags.split(' '))
   }
 
-  if (artifact) {
-    args.push('--artifact', `${earthfile}+${target}/`, `${artifactPath}`)
-  } else {
-    args.push(`${earthfile}+${target}`)
-  }
+  const targets = getTargetsFromEarthfile(target, earthfile)
```

Comment:
This should come from executing the CLI. It's probably a good time to make an improvement here and change the `target` input to be a list of targets instead of a single target. The CLI already supports passing multiple targets by specifying the `-t` flag multiple times.